lib/config/httpRoutes.js

```diff
@@ -48,6 +48,7 @@ module.exports = [
 
   {verb: 'get', url: '/:index/:collection', controller: 'document', action: 'search'},
   {verb: 'get', url: '/:index/:collection/:_id', controller: 'document', action: 'get'},
+  {verb: 'get', url: '/:index/:collection/_mGet/:_ids', controller: 'document', action: 'mGet'},
```

Comment:
Why is it `_ids` here and `ids` if put in the body part of the document?

```suggestion
  {verb: 'get', url: '/:index/:collection/_mGet/:ids', controller: 'document', action: 'mGet'},
```

Reply:
I finally changed to have the same URL, so it now is
`/:index/:collection/_mGet/` like the POST route.
Because in the SDK, the shortest route is the default one, and if they have the same size, it choose the POST route (i change to the GET to be default in this PR https://github.com/kuzzleio/sdk-javascript/pull/465)

Reply:
good change: ids should be passed as a querystring
wrong reason: SDK limitations should never influence core architecture choices